package.json

```diff
@@ -34,21 +47,21 @@
     "author": "Jesse Alama <jesse@igalia.com>",
     "license": "ISC",
     "devDependencies": {
-        "@types/jest": "^29.5.1",
+        "@rollup/plugin-typescript": "^11.1.6",
+        "@types/jest": "^29.5.12",
         "c8": "^9.1.0",
-        "jest": "^29.5.0",
+        "jest": "^29.7.0",
         "jest-light-runner": "^0.6.0",
         "prettier": "^3.0.0",
-        "ts-jest": "^29.1.0",
+        "rollup": "^4.14.1",
+        "ts-jest": "^29.1.2",
+        "tslib": "^2.6.2",
```

Comment:
I don't see why we need tslib since `importHelpers` aren't used?

Reply:
likely a mistake, PR raised by Jesse in #121 no longer has this